src/site/content/en/blog/declarative-link-capturing/index.md

```diff
@@ -0,0 +1,222 @@
+  them. In a desktop environment, it often makes sense to have multiple application windows open at
+  a time.
+- Single-window PWAs where the developer prefers to only have a single instance of the app open at
+  any time, with new navigations focusing the existing instance. Sub-use cases include:
+  - Apps that generally only make sense to have one instance running (e.g., a music player, a game).
+  - Apps that include multi-document management within a single instance (e.g., an HTML-implemented
+    tab strip).
+
+## Current status {: #status }
+
+<div class="w-table-wrapper">
+
+| Step                                     | Status                               |
+| ---------------------------------------- | ------------------------------------ |
+| 1. Create explainer                      | [Complete][explainer]                |
+| 2. Create initial draft of specification | [In Progress][spec]                  |
+| 3. Gather feedback & iterate on design   | [In progress](#feedback)             |
+| **4. Origin trial**                      | **[In progress][ot]** (on Chrome OS) |
+| 5. Launch                                | Not started                          |
+
+</div>
+
+### Enabling via about://flags
+
+To experiment with Declarative Link Capturing locally, without an origin trial token, enable the
+`#enable-desktop-pwas-link-capturing ` flag in `about://flags`.
+
+### Enabling support during the origin trial phase
+
+Starting in Chromium&nbsp;91, Declarative Link Capturing will be available as an origin trial in
+Chromium. The origin trial is expected to end in Chromium&nbsp;94 (October 13, 2021).
+
+{% Aside 'caution' %} The origin trial will initially be limited to the Chrome&nbsp;OS platform.
+{% endAside %}
+
+{% include 'content/origin-trials.njk' %}
+
+### Register for the origin trial {: #register-for-ot }
+
+{% include 'content/origin-trial-register.njk' %}
+
+## How to use Declarative Link Capturing? {: #use }
+
+Developers can declaratively determine how links should be captured by leveraging the additional Web
+Application Manifest field `"capture_links"`. It takes a string or a an array of strings as its
+value. If an array of strings is given, the user agent chooses the first supported item in the list,
+defaulting to `"none"`. The possible values dictate the link capturing behavior as follows:
+
+- `"none"` (the default): No link capturing; links clicked leading to this PWA scope navigate as
+  normal without opening a PWA window.
+- `"new-client"`: Each link clicked leading to this PWA scope opens a new PWA window at that URL.
+- `"existing-client-navigate"`: When a link is clicked leading to this PWA scope, the user agent
+  finds an existing PWA window, and causes it to navigate to the opened URL. If more than one PWA
+  window exists, the browser may choose one arbitrarily. This behaves like `"new-client"` if no
+  window is currently open. 🚨 Careful! This option potentially leads to data loss, as pages can be
```

Comment:
I don't know what this emoji is. It looks like a fez with a star on it.

Reply:
It's a police car light. https://emojipedia.org/police-car-light/

Reply:
They haven't looked like that on American police cars in like half a century. No wonder I didn't recognize it.